id_minter/src/main/scala/uk/ac/wellcome/platform/idminter/database/IdentifiersDao.scala

```diff
@@ -16,22 +16,22 @@ class IdentifiersDao @Inject()(db: DB, identifiers: IdentifiersTable)
 
   implicit val session = AutoSession(db.settingsProvider)
 
-  def findSourceIdInDb(miroId: String): Future[Option[Identifier]] =
+  def lookupMiroID(miroID: String): Future[Option[Identifier]] =
```

Comment:
Makes sense to have this be more specific as this only looks for a single thing, but worth thinking about (if not implementing) how this will work with multiple identifiers.